therr-services/maps-service/src/handlers/spaceMetrics.ts

```diff
@@ -61,6 +61,14 @@ const createSpaceMetric = async (req, res) => {
         .catch((err) => handleHttpError({ err, res, message: 'SQL:SPACES_ROUTES:ERROR' }));
 };
 
+async function getMetrics(startDate, endDate, spaceId) {
+    const range = endDate - startDate;
+    const currentSeries = await Store.spaceMetrics.getForDateRange(startDate, endDate, { spaceId });
+    const previousSeries = await Store.spaceMetrics.getForDateRange(startDate - range, endDate - range, { spaceId });
+
```

Comment:
It would be technically faster to perform these requests simultaneously like so:

```
const currentSeriesPromise = Store.spaceMetrics.getForDateRange(startDate, endDate, { spaceId });
const prevSeriesPromise = Store.spaceMetrics.getForDateRange(startDate - range, endDate - range, { spaceId });

return Promise.all([currentSeriesPromise, prevSeriesPromise])
```

Async/Await is blocking whereas Promise.all is concurrent